app/lib/bk/compat/parsers/gha/expressions.rb

```diff
@@ -138,6 +138,11 @@ def self.replace_context_vars(var_name)
         "$#{var_name}"
       end
 
+    def self.replace_context_secrets(var_name)
```

Comment:
Small 2col indent for this, otherwise quick and easy 👍

Reply:
oh i think there mustn't be consistency in this project or something 🤷‍♂️ 
my editor indents things all wonky all the time